README.md

```diff
@@ -1,85 +1,52 @@
 # Nicotine+
 
-[![N|Solid](files/icons/96x96/nicotine-plus.png)](https://github.com/Nicotine-Plus/nicotine-plus/)
+Nicotine+ is a graphical client for the [Soulseek](https://www.slsknet.org/) peer-to-peer system. It is an attempt to keep [Nicotine](https://web.archive.org/web/20150720173459/http://nicotine.thegraveyard.org/) working with the latest libraries, kill bugs, keep current with the Soulseek protocol, and add some new features that users want and/or need.
 
-## Intro
+# Download Nicotine+
+The current stable version of Nicotine+ is 1.4.1
 
-A graphical client for the SoulSeek peer-to-peer system.
-
-Nicotine+ is a graphical client for the SoulSeek peer-to-peer system. It is an attempt to keep Nicotine working with the latest libraries, kill bugs, keep current with the SoulSeek protocol, and add some new features that users want and/or need.
-
-# License
-
-Nicotine+ released under the terms of the [GNU Public License v3](https://www.gnu.org/licenses/gpl-3.0-standalone.html) or later.
-
-# Getting Involved
-Please come and join us in the `#nicotine+` channel on Freenode!
-
-If you'd like to contribute, you have a couple of options to get started. You can open an issue ticket on GitHub, discuss in `#nicotine+`, or post to the project [mailing list](mailto:nicotine-team@lists.launchpad.net). Developers are also encouraged to join the [Launchpad Team](https://launchpad.net/~nicotine-team) or subscribe to the mailing list so that they are automatically notified of failed commits.
-
-There is a current list of things [TODO](doc/TODO.md). If you'd like to translate Nicotine+ into another language it has not been already, see [TRANSLATIONS](doc/TRANSLATIONS.md).
-
-You want to contact someone? See: [MAINTAINERS](AUTHORS.md)
-
-# Precompiled Packages
+## Linux
```

Comment:
Please change this to GNU/Linux. Linux is [just a kernel](https://www.gnu.org/gnu/linux-and-gnu.html), after all.